src/services/user.services.js

```diff
@@ -27,6 +27,11 @@ const userServices = {
         user.id = newId;
         userData.push(user);
         return user;
+    },
+
+    login (email) {
+        const foundUser = userData.find(user => user.email === email);
+        return foundUser || {};
```

Comment:
Watch this video to learn how to implement jet for login: https://youtu.be/mbsmsi7l3r4